package.json

```diff
@@ -33,6 +33,7 @@
     "react-scripts": "5.0.0",
     "react-shimmer-effect": "^1.0.9",
     "react-twitter-widgets": "^1.11.0",
+    "react-simple-maps": "^2.3.0",
```

Comment:
Can you mention this library in your play's readme like the name, link, and what it does?